src/Sentry.AspNetCore/SentryHostBuilderExtensions.cs

```diff
@@ -0,0 +1,93 @@
+using System;
+using System.ComponentModel;
+using Microsoft.Extensions.DependencyInjection;
+using Sentry.AspNetCore;
+
+// ReSharper disable once CheckNamespace
+namespace Microsoft.Extensions.Hosting
+{
+    /// <summary>
+    /// Extension methods to <see cref="IHostBuilder"/>
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public static class SentryHostBuilderExtensions
+    {
+        /// <summary>
+        /// Uses Sentry integration.
+        /// </summary>
+        /// <param name="builder">The builder.</param>
+        /// <returns></returns>
+        public static IHostBuilder UseSentry(this IHostBuilder builder)
```

Comment:
`IHostBuilder` lives in `Microsoft.Extensions.Hosting` so this could be added to `Sentry.Extensions.Logging`.

Reply:
I would've personally kept it in the asp.net core part, as it is the replacement of WebHostBuilder in the future.
Would you like me to move it ?